### PR TITLE
ECIP-1093: Change the ETC Proof of Work Algorithm to RandomX

### PR DESCRIPTION
**Pull request by gitr0n1n**
- ECIP to utilize RandomX

### CODE REVIEW

**Comment by TheEnthusiasticAs** [comment]
My first review refer only to this 
image

File: _specs/ecip-randomX.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+lang: en
+ecip: To Be Determined
```
Comment (realcodywburns) [inline _specs/ecip-randomX.md:3]: please update to reflect ecip-1093

**Comment by developerkevin** [comment]
This proposal garnered a lot of negativity and unfavorable comments in last week's call. Also, there was nobody from to champion this "asic resistant" proposal on the call. We agreed in that meeting it will not be followed up on.

Can we close this ECIP?

**Comment by q9f** [comment]
the author can abandon or withdraw it, but to reject it, we would need community consensus. feel free to put it on the agenda.

**Comment by gitr0n1n** [comment]
> This proposal garnered a lot of negativity and unfavorable comments in last week's call. Also, there was nobody from to champion this "asic resistant" proposal on the call. We agreed in that meeting it will not be followed up on.
> 
> Can we close this ECIP?

As the author of this ECIP, I was never contacted regarding the ECIP I submitted a mere 3 weeks ago. Is there any recording of this? I was under the impression the referenced CDC was a disaster. I saw a video where Charles mentioned there wasn't productive conversation about this ECIP in the meeting.
https://www.youtube.com/watch?v=F0lR_u7BVho&feature=youtu.be&t=490

And I saw a 2Miners article where they are against the ASIC options:
https://2miners.com/blog/ethereum-classic-future-discussing-etc-network-current-state-and-security-plan/

**Comment by gitr0n1n** [comment]
> the author can abandon or withdraw it, but to reject it, we would need community consensus. feel free to put it on the agenda.

I listened to the call and have clarity on the community's opinions. Thanks for the recording and transparency. Do I need to do anything at this point @q9f ?

8/28 CDC:
https://www.youtube.com/watch?v=4W1l5krLPqI

**Pull request merged**